pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -19,10 +19,11 @@ package elb
 import (
 	"fmt"
 	"reflect"
-	"sigs.k8s.io/cluster-api-provider-aws/pkg/record"
 	"strings"
 	"time"
 
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/record"
+
```

Comment:
nit: remove space here and resort to group with the imports below

Reply:
changed